### ISSUE TITLE
Issue with buffer bindings for multiple SBOs

### ISSUE THREAD

**Issue by sungiant** [KhronosGroup/MoltenVK#959]
Hi, I'm having trouble with bindings for compute shaders that use multiple storage buffers, perhaps I've found an issue?

I consistently get the following type of error when queueing up work from compute pipelines with descriptor set layouts and associated shader modules that use more than one storage buffer:

```
validateComputeFunctionArguments:762: failed assertion `Compute Function(main0): missing buffer binding at index 5 for spvBufferSizeConstants[0].'
```
- I'm using OSX and MoltenVK.
- I've tested on multiple machines.
- On Windows with Vulkan the same examples do not have issues.
- Being concerned that I had something wrong with API usage in my test case I created another test case based on [this](https://github.com/SaschaWillems/Vulkan/blob/master/examples/computeraytracing/computeraytracing.cpp) example - I continue to experience the same kind of error with MoltenVK.

Any help would be much appreciated.  Thanks.

**Comment by billhollings** [comment]
This is fixed in PR #979. Please retest and close this issue if it fixes it for you.

**Comment by sungiant** [comment]
Brilliant, thank you.  I've just tested my project with these changes and the issue looks to be resolved.
